src/main/docs/guide/ssl.adoc

```diff
@@ -0,0 +1,33 @@
+The Cassandra DataStax Java Driver includes https://docs.datastax.com/en/developer/java-driver/latest/manual/core/ssl/#driver-configuration[support for SSL], to secure traffic between the driver and Cassandra.
+
+This an example of a configuration for Micronaut Cassandra that enables SSL connections. The relevant values are defined under the `cassandra.\*.advanced.ssl-engine-factory.*` property key and documented further in the https://docs.datastax.com/en/developer/java-driver/latest/manual/core/configuration/reference/[Datastax Configuration Reference].
+
+[configuration]
+----
+cassandra:
+  default:
+    advanced:
+      ssl-engine-factory:
+        class: DefaultSslEngineFactory
+        cipher-suites:
+          - TLS_RSA_WITH_AES_128_CBC_SHA
+          - TLS_RSA_WITH_AES_256_CBC_SHA
+        trust-store-path: <path to trust store>
+        trust-store-password: 12345
+        key-store-path: <path to key store>
+        key-store-password: 12345
+      metadata:
+        schema:
+          enabled: false
+    basic:
+      contact-points:
+        - "127.0.0.1:9042"
+        - "127.0.0.2:8042"
+      load-balancing-policy:
+        local-datacenter: datacenter1
+----
+
+- class: The class of the factory. `DefaultSslEngineFactory` is the default built-in implementation
+- cipher-suites: cipher suites to enable when creating a connection
+- trust-store and key-store properties: these are optional and if not present the system property configuration is used instead
```

Comment:
Make sure these are spelled correctly as per the docs.  keystore isn't required either in the tests, so I'd remove it